2_phase_minting_schedule/main.tex

```diff
@@ -4,61 +4,62 @@
+The second section introduces reasoning for the two-phase model pertaining to network demand, the protocol and node operation, the third introduces an empirical analysis of existing networks, and the final section derives parameters for the new minting schedule.
 
 \section{Motivation \& Rationale}
 
-\textit{Demand-oriented rationale}
-\\\\
-The following premises form the basis for this proposal: 
+\subsection{Demand uncertainty}
+
+These demand-related premises form the basis for the two-phase model: 
 \begin{enumerate}
 \item There is little certainty regarding the length of time that will elapse between the network's launch and the first trickle of demand from customers (developers, end-users \& others).
 \item This early-stage demand is particularly fragile, because new customers are naturally more fickle. Specifically, neither developers nor their end-users have compelling reasons to tolerate an inadequate service (e.g. too few stakers available to service a sharing policy), such as the sunk costs of integration, reliance on the service, or other forms of path dependency, all of which are more likely to play a role later in the network's lifetime. New customers also take a greater risk than later adopters, as they do not have the experience of existing customers as evidence of a reliable or valuable service.
-\item Over the first 5 years of the network's existence, the previous draft model would see the the nominal inflation rate drop over 5 times (to roughly 17.6\%). If the fee market has not sufficiently matured by this point, this reduction in earnings may render some node operations unsustainable. 
-\item At least some early customers will inflexibly require long-lasting sharing policies. This means that the protocol must motivate the successive re-commitment to long token lock durations (i.e. staking for 12+ months) by a sufficient population of node operators. A diminished inflation rate, in lieu of a mature fee market, may not be enough to incentivize this repeated and essential re-commitment. 
+\item A popular subsidy model involves daily token issuance dropping exponentially, right from launch. If the fee market does not sufficiently mature in time to make up for the sharp reduction in earnings the early years of the network's existence, some node operations may be rendered entirely unsustainable. 
+\item Most importantly, some of NuCypher's early customers will (inflexibly) require long-lasting sharing policies. This means that the protocol must motivate the successive re-commitment to long token lock durations (i.e. staking for 12+ months) by a sufficient population of node operators. A diminished subsidy issuance, in lieu of a mature fee market, may not be enough to incentivize this repeated and essential re-commitment. 
```

Comment:
```suggestion
\item Most importantly, some of NuCypher's early customers may (inflexibly) require long-lasting sharing policies. This means that the protocol must motivate the successive re-commitment to long token lock durations (i.e. staking for 12+ months) by a sufficient population of node operators. A diminished subsidy issuance, in lieu of a mature fee market, may not be enough to incentivize this repeated and essential re-commitment. 
```

Reply:
^ Ignore above, it was an old change, before you made updates based on maclane's suggestions.